src/operator/slice_channel-inl.h

```diff
@@ -62,11 +67,63 @@ struct SliceChannelParam : public dmlc::Parameter<SliceChannelParam> {
   }
 };  // struct SliceChannelParam
 
+struct SplitKernel {
+  template<typename DType>
+  static MSHADOW_XINLINE void Map(size_t i,
+                                  const DType *in_data, DType** out_data, const size_t* indices,
+                                  size_t num_sections, size_t axis_size, size_t trailing_size) {
+    // printf("%lu\n", i);
+    size_t idx = i / trailing_size % axis_size;
```

Comment:
parenthesis missing

Reply:
Is considered bad practice to rely on operator precedence when you can make code more readable adding parenthesis. This is stated in many reference programming books. You can also do "man operator" but that's not the point. http://wiki.c2.com/?OperatorPrecedenceConsideredHarmful

Reply:
How is this not maintainable and not readable? Would you do something like
```c++
int a = (b + c) - d;
```
?

Reply:
size_t idx = (i / trailing_size) % axis_size;
is it so expensive to add to parens?

For + - the associativity doesn't matter much except overflow right?

Reply:
Then why is it so necessary to add the parenthesis...? It's only making the code look very redundant. I actually expected more insights from this community's code reviews rather than style checks/nitpicking...